asm/internal/ll.bnf

```diff
@@ -1369,7 +1371,12 @@ SelectInst
 // ~~~ [ call ] ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 CallInst
-	: "call" FastMathFlags ParamAttrs Type Value "(" Args ")" FuncAttrs   << astx.NewCallInst($3, $4, $6) >>
+	: OptTail "call" FastMathFlags OptCallConv ParamAttrs Type Value "(" Args ")" FuncAttrs CommaAttachedMDs   << astx.NewCallInst($5, $6, $8) >>
```

Comment:
Thanks! The annotation for calling convention will come in use later by decomp! At that point we can propagate the data along to the IR layer through `astx.NewCallInst`.